### PR TITLE
chore: moving docs repo into the monorepo

### PR DESCRIPTION
**Pull request by 0xisk**
## Explanation

This PR is for moving Docs mdBook inside the monorepo. The `gh-pages.yml` workflow I think will need some fixes.

## Related Issues

N/A

## Screenshots

N/A

## Manual Testing Steps

N/A

## Pre-Merge Checklist

- [x] PR template is filled out
- [x] Pre-commit and pre-push hook checks are passed
- [ ] E2E tests are passed locally
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [ ] PR is linked to the appropriate GitHub issue
- [ ] PR has been added to the appropriate release Milestone

> PR template source from [github.com/MetaMask](https://github.com/MetaMask)

### CODE REVIEW

File: .github/workflows/gh-pages.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      MDBOOK_VERSION: 0.4.21
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup mdBook
+        uses: peaceiris/actions-mdbook@v1
```
Comment (0xmad) [inline .github/workflows/gh-pages.yml:24]: This action won't run because it's not from us, github or verified authors.

File: docs/book.toml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[book]
+authors = ["AtHeartEngineer", "0xIsk", "0xmad"]
+language = "en"
+multilingual = false
+src = "src"
+title = "CryptKeeper Documentation"
+
+[build]
+create-missing = true
```
Comment (0xmad) [inline docs/book.toml:9]: Need an empty line at the end

File: docs/src/guides/contributing.md
Comment (0xmad) [inline docs/src/guides/contributing.md:1]: This one is outdated

File: docs/src/guides/integration/connect-to-extension.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# 🔌 Connect to extension
+
+## Connect to extension
+
+Use `zkpr.connect` to access the extension.
```
Comment (0xmad) [inline docs/src/guides/integration/connect-to-extension.md:5]: We don't have `zkpr` anymore

File: docs/src/guides/integration/create-identity.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                    await this.identityService.insert(identity)
+
+                    return true
+                } catch (error: any) {
+                    console.log("CREATE_IDENTITY: Error", error);
+                    throw new Error(error.message)
+                }
+            }
+        )
+```
```
Comment (0xmad) [inline docs/src/guides/integration/create-identity.md:50]: I don't think users need this code and documentation how it works under the hood.

File: docs/src/guides/integration/generate-proof.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+```typescript
+async function semaphoreProof(
+    externalNullifier: string,
+    signal: string,
+    circuitFilePath: string,
+    zkeyFilePath: string,
+    merkleProofArtifactsOrStorageAddress: string | MerkleProofArtifacts,
+    merkleProof?: MerkleProof
+) {
+```
```
Comment (0xmad) [inline docs/src/guides/integration/generate-proof.md:16]: This is outdated method signature

File: docs/src/guides/integration/generate-proof.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+Any circuit files (`circuitFilePath` and `zkeyFilePath`) can be used, but ready-to-use and audited circuit files can be found [here](http://www.trusted-setup-pse.org/) and [here](https://drive.google.com/file/d/1Yi14jwly70VwMSuqJrPCc3j15MWeE7mc/view?usp=sharing).
+
+Merkle witness or Merkle tree components (`merkleProofArtifactsOrStorageAddress` can be provided by a smart contract or as a structured input:&#x20;
+
+* `StorageAddress` points to a Merkle tree on-chain
+* `merkleProofArtifact` is a Merkle tree in a specific format
+  * [@src/types/index.ts](https://github.com/privacy-scaling-explorations/crypt-keeper-extension/blob/817ec0e1f336ab61c9c70f4918853e7c279dd21d/src/types/index.ts#L56)
+  * ```typescript
+    export type MerkleProofArtifacts = {
```
Comment (0xmad) [inline docs/src/guides/integration/generate-proof.md:30]: We use interfaces

File: docs/src/guides/integration/inject-client.md
Comment (0xmad) [inline docs/src/guides/integration/inject-client.md:1]: This API is outdated

File: docs/src/guides/integration/zkitter-examples.md
Comment (0xmad) [inline docs/src/guides/integration/zkitter-examples.md:1]: Zkitter support is dropped and for examples we have demo folder

**Comment by 0xisk** [comment]
Thanks @0xmad for your review. I will work on the docs internally and on fixing your comments on anther PR. In this PR I am moving the Docs into the monorepo.

**Review by 0xmad — APPROVED** [review]
@0xisk thanks!

**Pull request merged**